app/src/main/java/de/hbch/traewelling/ui/checkIn/EditCheckInFragment.kt

```diff
@@ -70,6 +85,28 @@ class EditCheckInFragment : AbstractCheckInFragment() {
                 )
                 alertBottomSheet.show(parentFragmentManager, AlertBottomSheet.TAG)
             }
-        })
+        }
+        if (!args.replace) {
```

Comment:
Is this `replace` arg even needed now? If yes, I guess the name is a bit misleading because I don't understand what it does 😅

Reply:
We still have two different api requests depending on what we do, so i renamed it to `changeDestination` as we don't need to send a destination if we only update the status body